examples/bri-3/src/bri/communication/api/messages.controller.spec.ts

```diff
@@ -273,34 +300,28 @@ describe('MessageController', () => {
 
     it('should perform the update if existing id passed', async () => {
       // Arrange
-      const createRequestDto = {
-        id: '123',
-        from: existingBpiSubject1.id,
-        to: existingBpiSubject2.id,
-        content: 'hello world',
-        signature:
-          '0xb27e0845a034ae61be153bf305985e4c66e9e0b0009289c764eceeb9d886a33b435cef57834b078c3eca85e015374c6f8e1406c3ac6b13144a98f794ba7c56ce1c',
-        type: 1,
-      } as CreateBpiMessageDto;
-
-      const newBpiMessageId = await mController.createBpiMessage(
-        createRequestDto,
+      messageStorageAgentMock.getBpiMessageById.mockResolvedValueOnce(
+        existingBpiMessage,
       );
 
       const updateRequestDto = {
         content: 'this is content2',
         signature: 'xyz2',
       } as UpdateBpiMessageDto;
+      messageStorageAgentMock.updateBpiMessage.mockResolvedValueOnce({
+        ...existingBpiMessage,
+        content: updateRequestDto.content,
+        signature: updateRequestDto.signature,
+      } as BpiMessage);
 
       // Act
-      await mController.updateBpiMessage(newBpiMessageId, updateRequestDto);
-
-      // Assert
-      const updatedBpiMessage = await mController.getBpiMessageById(
-        newBpiMessageId,
+      const updatedBpiMessage = await mController.updateBpiMessage(
+        existingBpiMessage.id,
+        updateRequestDto,
       );
 
-      expect(updatedBpiMessage.id).toEqual(newBpiMessageId);
+      // Assert
+      expect(updatedBpiMessage.id).toEqual(existingBpiMessage.id);
```

Comment:
similar to last PR, please use assertion on the complete object.

Reply:
i dont think it worked here but can check

Reply:
not the same, returned here is dto, and existing bpi message is model